codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/customizations/HttpChecksumRequiredGenerator.kt

```diff
@@ -38,7 +39,7 @@ class HttpChecksumRequiredGenerator(
             is OperationSection.AdditionalRuntimePlugins -> writable {
                 section.addOperationRuntimePlugin(this) {
                     rustTemplate(
-                        "#{HttpChecksumRequiredRuntimePlugin}::new()",
```

Comment:
would you ever not want a shared runtime plugin? Should we add an impl Into or something to make this (and other additional plugin code) cleaner?

we already have `addOperationRuntimePlugin` so you could put the wrapping there

Reply:
My thought on this was having the flexibility to actually share runtime plugins. Making it wrap them would make this impossible without a later refactor.

Reply:
All made easier in 01720a953b12fc775e3f0c37d83d6f6887bba4b8